core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -261,6 +282,24 @@ class GroupCoordinator(val brokerId: Int,
     }
   }
 
+  /**
+    * Decide whether the group should be transiting (or continue) to be at PrepareRebalance.
+    */
+  private def maybeRebalanceOnKnownMemberRejoin(group: GroupMetadata,
```

Comment:
good call with splitting this logic into a method! 👍

Reply:
lol